src/main/webapp/js/add-location-loader.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
`add-location`

Reply:
do you want me to rename the file to be add-location.js?